packages/fxa-auth-server/lib/routes/oauth.js

```diff
@@ -197,6 +197,7 @@ module.exports = (log, config, oauthdb, db, mailer, devices) => {
           const newSessionToken = await db.createSessionToken(
             sessionTokenOptions
           );
+          grant.session_token_id = newSessionToken.id;
```

Comment:
should probably rename this or something, otherwise the id is modified on the fly

Reply:
Is `session_token_id` used in oauthRouteUtils.newTokenNotification? Otherwise this is confusing because the value is deleted [down here](https://github.com/mozilla/fxa/pull/2930/files#diff-ba2df0b1af69d2687f589b68e4d42930R218).

Also, some places in the grant make use of session_token_id, some [sessionTokenId](https://github.com/mozilla/fxa/blob/bc1a15c98cae7932c5aadf00c42bdc068cd5c31f/packages/fxa-auth-server/fxa-oauth-server/lib/grant.js#L173). Within grants, we should probably choose one variant and stick with it to avoid the discrepancy biting us.

Reply:
> Is session_token_id used in oauthRouteUtils.newTokenNotification? Otherwise this is confusing because the value is deleted down here.

yea exactly